src/arch/linux.rs

```diff
@@ -16,3 +17,11 @@ pub fn package_installed(package: &str) -> Result<bool> {
 		None
 	))
 }
+
+/// Check if file permissions are secured
+pub fn file_permissions(file: &str, perms: &str) -> Result<bool> {
+	let f = File::open(&file)?;
+	let metadata = f.metadata()?;
+	let current_perms = metadata.permissions().mode();
+	Ok((current_perms & 0o7777) == u32::from_str_radix(&perms, 8)?)
```

Comment:
tbh i'd prefer less intermediary stuff here, just have the file handle as one variable and the rest can go inside Ok lol